Cart/Controllers/CartController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Cart.Business.Interfaces;
+using Cart.Mappers;
+using Cart.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Cart.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class CartController : ControllerBase
+    {
+        private readonly ICartingService _cartingService;
+
+        public CartController(ICartingService cartingService)
+        {
+            _cartingService = cartingService ?? throw new ArgumentNullException(nameof(cartingService));
```

Comment:
Usually, constructor shouldn't throw an exceptions. It just should initialize the service

```suggestion
            _cartingService = cartingService;
```